src/Pos/Wallet/Web/Api.hs

```diff
@@ -104,8 +137,8 @@ type WalletApi =
      :> "txs"
      :> "payments"
      :> Capture "passphrase" CPassPhrase
-     :> Capture "from" CAddress
-     :> Capture "to" CAddress
+     :> Capture "from" CAccountAddress
+     :> Capture "to" CAccountAddress
```

Comment:
@flyingleafe is correct. @Martoon-00 yes I can do that.

Will do detailed review today